### DISCUSSION TITLE
Spring Boot 3 compatible

### DISCUSSION THREAD

**Discussion by jimartinmoreno** [MarcGiffing/bucket4j-spring-boot-starter#129] [Q&A] [answered]
Hi,
We are planning on migrating our services to Spring Boot 3. Is there an estimation of when is it going to be released a new release Spring Boot 3 compatible?

Thank you in advance and Merry Christmas!

**Comment by k-rkr-e-nettet-dk** [comment]
We would also like to incoporate this feature again, after upgrading to spring boot 3.
I've just had a quick test run with 0.9.0-RC1, and it definately seems to be working great, but we would like to wait for 0.9.0 before upgrading.

So any estimate of when version 0.9.0 is avaiable would be greatly appretiated.

Thanks for a realy awesome library.

**Reply by MarcGiffing** [reply, marked as answer]
0.9.0 is released and will be available in  the next hours

**Reply by jimartinmoreno** [reply]
Amazing!
